### PR TITLE
'Extend' method works wrong for implementing inheritance in testcafe (close #322)

### LINKED ISSUE #322: 'Extend' method works wrong for implementing inheritance in testcafe
We need use 'extend' method from utils for this https://github.com/DevExpress/testcafe/blob/master/src/client/core/utils/service.js#L10

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @churkin @VasilyStrelyaev

### CODE REVIEW

File: test/client/fixtures/utils/extend-test.js [hunk 1]
```diff
@@ -81,3 +81,50 @@ test('target is build-in type', function () {
     strictEqual(result.property1, obj2.property1);
     strictEqual(Object.keys(result).length, 1);
 });
+
+test('second object has first object (infinite embedding)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/utils/extend-test.js]: maybe just
```
test('infinite embedding', function () {
```

Comment (inikulin) [inline test/client/fixtures/utils/extend-test.js]: Actually it's `circular dependency`

**Comment by churkin** [comment]
lgtm

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
FPR
@VasilyStrelyaev merge please

**Pull request merged**